### ISSUE TITLE
A team's table to be added below the cards on Team's page.

### ISSUE THREAD

**Issue by kaushalvivek** [gokulbnr/E-Cell-Website#12]
The cards will contain the core team members, while the table will have name, department and contact info of all members preferably organised by departments.

**Comment by Pk13055** [comment]
Working on this.

**Comment by Pk13055** [comment]
#20

**Comment by kaushalvivek** [comment]
The table has to render different members, not the same as the one being rendered on the cards.
New models will have to be created for the table. The basic is: Cards are for core members, table will display all members. They will not be the same. Also, instead of a column for Facebook, if you can have one for Social Media, and have a facebook as well as a LinkedIn button there, that would be perfect.

**Comment by Pk13055** [comment]
@kaushalvivek Okay, but give me the data names yeah? I won't be able to render anything unless I don't know what data I'm receiving. For now, I'll keep the table structure ready with the buttons and stuff using the existing data, you can raise another issue once the data model is ready and populated.

**Comment by kaushalvivek** [comment]
@Pk13055 Sure, I'll give you the data asap. Sorry, have been travelling, couldn't give much time to this.
It's looking good for now, the issues posted and some rudimentary work remains and we are good to go.

**Comment by Pk13055** [comment]
Open up a new issue for adding the data, basically just change one variable. Closing (and merging) for now.

**Comment by Pk13055** [comment]
#23
